README.md

```diff
@@ -130,7 +130,7 @@ mocha -R spec integration/regtest.js
 If any changes have been made to the bindings in the "src" directory, manually compile the Node.js bindings, as defined in `bindings.gyp`, you can run (-d for debug):
 
 ```bash
-$ node-gyp -d rebuild
+$ node-pre-gyp -d rebuild
```

Comment:
Also I think it would be good to document that `node-pre-gyp` can be installed via running `npm install node-pre-gyp -g` if it hasn't already been installed (before the first time it's needed).